modules/oe_theme_content_project/src/Plugin/ExtraField/Display/ProjectExtraFieldBase.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_project\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Base class for Project extra fields.
+ */
+abstract class ProjectExtraFieldBase extends ExtraFieldDisplayFormattedBase implements ContainerFactoryPluginInterface {
```

Comment:
Do we need this class at all? It seems that only percentage field uses it, and it even doesn't use the entityTypeManager service.

Reply:
Fixed : Class removed and class `PercentageExtraField `extends now `ExtraFieldDisplayFormattedBase`